test/system/executable/CMakeLists.txt

```diff
@@ -26,6 +26,12 @@ if(PYTHONINTERP_FOUND)
         simple_configuration
     )
 
+    # Whether one or other signal in test
+    set(SIGNAL_CONFIGURATIONS
+        "--sigterm"
+        ""
+    )
```

Comment:
I'll create two lists here, one for the argument and another for the test name, and try to iterate both at the same time. Check [this](https://stackoverflow.com/questions/7932205/parallel-iteration-over-lists-in-makefile-or-cmake-file) to do so.

Reply:
That is what I do but with two foreach. I do not understand what do you want